config/asgi.py

```diff
@@ -18,28 +18,30 @@
 # TODO: from channels.security.websocket import AllowedHostsOriginValidator
 from django.core.asgi import get_asgi_application
 
+logger = logging.getLogger(__name__)
+
 # Here we setup a debugger if this is desired. This obviously should not be run in production.
 debug = os.environ.get("DEBUG")
 if debug:
-    logging.info(f"Django is set to use a debugger. Provided debug mode: {debug}")
+    logger.info("Django is set to use a debugger. Provided debug mode: %s", debug)
```

Comment:
Is this a "bad practice to use f strings in a logger call" situation?

Reply:
Yeah, there are two issues here.

https://docs.astral.sh/ruff/rules/logging-f-string/
https://docs.astral.sh/ruff/rules/root-logger-call/